plugins/scaffolder/src/components/TemplatePage/TemplatePage.tsx

```diff
@@ -145,6 +145,11 @@ export const TemplatePage = () => {
   const { templateName } = useParams();
   const navigate = useNavigate();
   const rootLink = useRouteRef(rootRouteRef);
+  const fields = {
+    RepoUrlPicker,
+    OwnerPicker,
+    ...scaffolderApi.getRegisteredFields(),
```

Comment:
This isn't quite enough to make the fields fully customizable, which we'd want. The missing piece is the `createValidator` function above, where you can see a check for `'RepoUrlPicker'` specifically. As part of this solution we'd need to generalize that and allow these custom components to provide their own validation logic.

Reply:
Ohh I see I missed that!